### PR TITLE
Add slide about OSs

### PR DESCRIPTION
**Pull request by maxnoe**

### CODE REVIEW

File: intro/Makefile [hunk 1]
```diff
@@ -3,7 +3,10 @@ include ../common/document.mk
 
 FIGURES := $(addprefix build/figures/, $(addsuffix .pdf, os programming interest))
 
-build/intro.pdf: $(FIGURES) $(addprefix build/figures/, vim.png nano.png real_programmers.png code.png)
+build/intro.pdf: $(FIGURES) $(addprefix build/figures/, \
+	vim.png nano.png real_programmers.png code.png windows.png macos.png tux.jpg\
+	fedora.png archlinux.png ubuntu.png debian.png
```
Comment (aknierim) [inline intro/Makefile:8]: Missing `\` at end of line, change `debian.png` to `debian.pdf`

**Review by aknierim — CHANGES_REQUESTED** [review]

**Comment by chrbeckm** [comment]
The ci notes: ll 1547:
```
wget https://www.debian.org/logos/openlogo.pdf -O build/figures/debian-original.pdf
--2023-09-24 11:45:03--  https://www.debian.org/logos/openlogo.pdf
Resolving www.debian.org (www.debian.org)... 128.31.0.62, 2603:400a:ffff:bb8::801f:3e
Connecting to www.debian.org (www.debian.org)|128.31.0.62|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 4964 (4.8K) [application/pdf]
Saving to: ‘build/figures/debian-original.pdf’

     0K ....                                                  100% 1.32G=0s

2023-09-24 11:45:03 (1.32 GB/s) - ‘build/figures/debian-original.pdf’ saved [4964/4964]

Traceback (most recent call last):
  File "/home/runner/work/toolbox-workshop/toolbox-workshop/intro/../common/crop.py", line 18, in <module>
    s = open(in_file, "r").read()
        ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen codecs>", line 322, in decode
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe2 in position 10: invalid continuation byte
make[1]: *** [Makefile:44: build/figures/debian.pdf] Error 1
make[1]: Leaving directory '/home/runner/work/toolbox-workshop/toolbox-workshop/intro'
make: *** [common/recursive.mk:9: intro_build] Error 2
make: *** Waiting for unfinished jobs....
```

Did it work on your system?

**Comment by LuckyJosh** [comment]
The crop.py script takes a .crop textfile as input,
like this one:  latex/figures/formats-few.crop

**Comment by aknierim** [comment]
The initial problem was, that `pdfcrop` failed for some reason, which led to trying it with the `crop.py` script. The way I see it, we could also try [pdfCropMargins](https://pypi.org/project/pdfCropMargins/) or just download the debian logo as png or svg (the latter of which would require to convert it to e.g. pdf) instead of pdf.

**Comment by LuckyJosh** [comment]
The contens of the `debian.crop` file
should be:

```
build/figures/debian-original.pdf
1
260 189 108 144
```

Sorry, I cant add it my self in the moment

File: intro/content/intro.tex [hunk 2]
```diff
@@ -299,6 +299,12 @@ \section{Ablauf}
   \end{description}
 \end{frame}
 
+\begin{frame}{Motivation}
```
Comment (maxnoe) [inline intro/content/intro.tex:302]: Den hatte ich mit Absicht rausgenommen

**Review by aknierim — APPROVED** [review]
LGTM!

**Pull request merged**